### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by ChrisMM**
@dhakiki 
- Update root README.md
- Update content/stack_focus.md
- Update content/customize_your_environment.md
- Update content/editors/vim.md
- Update content/best_coding_practices.md
- Update content/mentorship_guide.md

### CODE REVIEW

File: content/best_coding_practices.md [hunk 1]
```diff
@@ -83,18 +87,22 @@ already been built makes very little sense, provides very little benefit to the
...
+These features are written in [Gherkin](https://cucumber.io/docs/reference), which is a Domain Specific Language that lets you describe software's behaviour without detailing how that behaviour is implemented: "specification by example" as it's called.
 A good set of feature tests for this task would be:
 
 ```gherkin
 Feature: Configuration Settings
 
+As a dog
+I want a Settings option
+So that I can set my privacy.
+
```
Comment (dhakiki) [inline content/best_coding_practices.md:105]: 😎 🐶

File: content/editors/vim.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[Home](../../) > [Customize a Productive Coding Environment](../customize_your_environment.md) > Vim
+Back to [Customize a Productive Coding Environment](../customize_your_environment.md)
```
Comment (dhakiki) [inline content/editors/vim.md]: I actually liked having the breadcrumb in there so users can navigate where they please. was trying to improve usability! Access to Home seems valuable to me still.

**Review by dhakiki — COMMENTED** [review]
this is awesome! 🙌 thank you so much!!! just one comment on restoring the breadcrumb for usability sake but this looks great otherwise. thanks for contributing!

**Comment by ChrisMM** [comment]
All set. 🚀

**Comment by dhakiki** [comment]
Perfect! Merging.

**Pull request merged**